### PR TITLE
Updated readme.md

### PR DESCRIPTION
**Pull request by jagottsicher**
Minor changes. Typos and grammar corrected.

### CODE REVIEW

**Comment by jagottsicher** [comment]
Updated also eclair-cli, but if cecking jq then checking curl should be a help for beginners.

File: README.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ java -Dlogback.configurationFile=/path/to/logback-custom.xml -jar eclair-node-gu
  ------------- |----------------------------------------------------------------------------------------|-----------------------------------------------------------
   getinfo      |                                                                                        | return basic node information (id, chain hash, current block height) 
   connect      | nodeId, host, port                                                                     | open a secure connection to a lightning node
-  connect      | uri                                                                                    | open a secure connection to a lightning node
+  connect      | url                                                                                    | open a secure connection to a lightning node
```
Comment (sstone) [inline README.md]: this not a typo, it's a `uri` not `url`

File: README.md [hunk 2]
```diff
@@ -209,7 +209,7 @@ addresstype=p2sh-segwit
 deprecatedrpc=signrawtransaction
 ```
 
-You may also want to take advantage of the new configuration sections in `bitcoin.conf` to manage parameters that are network speficic, so you can reasliy run your bitcoin node on both mainnet and testnet. For example you could use:
+You may also want to take advantage of the new configuration sections in `bitcoin.conf` to manage parameters that are network specific, so you can easliy run your bitcoin node on both mainnet and testnet. For example you could use:
```
Comment (sstone) [inline README.md]: typo: `easliy` instead of `easily` :)

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](LICENSE)
 [![Gitter chat](https://img.shields.io/badge/chat-on%20gitter-red.svg)](https://gitter.im/ACINQ/eclair)
 
-**Eclair** (french for Lightning) is a scala implementation of the Lightning Network. It can run with or without a GUI, and a JSON-RPC API is also available.
+**Eclair** (French for Lightning) is a scala implementation of the Lightning Network. It can run with or without a GUI, and a JSON-RPC API is also available.
```
Comment (rorp) [inline README.md]: And it's Scala with capital S.

Comment (jagottsicher) [reply README.md]: True- added.

File: README.md (hunk 1 above)
Comment (jagottsicher) [reply README.md]: my bad.

File: README.md (hunk 2 above)
Comment (jagottsicher) [reply README.md]: Too funny, but corrected now.

**Comment by jagottsicher** [comment]
Executed requested changes.

**Pull request merged**